### ISSUE TITLE
sticky cookie not sent to /info

### ISSUE THREAD

**Issue by bryanlarsen** [sockjs/sockjs-client#269]
Is there any way to get cookies sent on the initial /info request so that the AWS Load Balancer knows which server sockjs is listening on?   We use the jsessionid support in sockjs-node;  that ensures that the websocket stays on the same server, however it doesn't ensure that the websocket stays on the same server as the preceding HTTPS request we used to set up the websocket.

We fork a new process for every websocket, so we use an HTTPS request to fork the process and send back a WSS URL that connects to the new process.   The HTTPS request contains an AWSELB cookie that the load balancer needs.

**Comment by brycekahle** [comment]
Do you mean setting a cookie on the `/info` response?

**Comment by bryanlarsen** [comment]
No, I mean sending the cookie from the browser to the server on the initial /info request.   The initial HTTPS request to our application code is setting the cookie.

Diving in further, it appears that sockjs-client sends the cookie when the request is to the same server as the page is on.   However, we need the websockets behind a different load balancer than the main server.    The main server can handle thousands of users per machine; each websocket server can only handle about 3 before bogging down...

To make it clearer:

The page is on example.com.   The code then makes a normal HTTPS request to farm.example.com, which is behind Amazon's load balancer.   Let's say that goes to machine farm1.   That machine then forks a new sockjs-node process, and sends back a URL that connects to that process.  The Amazon load balancer adds a Set-Cookie header to that response.    The client code then uses sockjs-client to attempt to connect to that new process, but it doesn't work because Amazon has sent the request to farm2 because the cookie is missing.

But before you spend too much effort, give me some more time to ensure that the problem isn't on my end, or at least ensure that I fully understand everything that's going on.  Just after I opened the bug I noticed some other strange things going on in the system, and I want to track them down before blaming your code.   Sorry for opening it early.
